website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -39,7 +39,7 @@
                     <td>
                         Now that you've shared your ${reviewable.provider.preprint_word}, take advantage of more OSF features:
                         <ul>
-                            % if has_psyarxiv_chronos_text:
+                            % if has_chronos_text:
                                 <li>Submit your preprint to an APA-published journal. <a href="https://help.osf.io/hc/en-us/articles/360044820913-Submit-to-APA-Journals-via-PsyArXiv">Learn more</a></li>
```

Comment:
The wording on this needs to change to "Submit your preprint to a journal." There's a chance that link might change as well, though I don't have a new link for you yet.

Reply:
The url should be https://help.osf.io/hc/en-us/articles/360044820913-submit-to-journal